src/codegeneration/ForOfTransformer.js

```diff
@@ -20,19 +23,25 @@ import {
   createVariableStatement
 } from './ParseTreeFactory.js';
 import {parseStatement} from './PlaceholderParser.js';
+import {LabelledStatement} from '../syntax/trees/ParseTrees.js';
 
 /**
  * Desugars for-of statement.
  */
-export class ForOfTransformer extends TempVarTransformer {
+export class ForOfTransformer extends TempVarTransformer {  
   /**
    * @param {ForOfStatement} original
+   * @param {Array.<LabelledStatement>=} labelSet
    * @return {ParseTree}
-   */
-  transformForOfStatement(original) {
+   */   
+  transformForOfStatement(original, labelSet = []) {
```

Comment:
This is a bit strange. The transformTreeType methods only take one parameter. Can you refactor so that that continues to be the case?